apps/nextjs/package.json

```diff
@@ -27,18 +27,17 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "superjson": "1.9.1",
+    "@types/node": "^18.15.11",
+    "@types/react": "^18.0.31",
+    "tailwindcss": "^3.3.1",
+    "postcss": "^8.4.21",
+    "autoprefixer": "^10.4.14",
```

Comment:
im no docker expert but why do these need to be production deps all of a sudden?

Reply:
when you run `pnpm install --prod `it only installs `dependencies` and when nextjs is trying to create to build it fails because it looks for those dependencies.

Reply:
imo just remove `--prod`, revert to `output: "standalone"`.